src/vs/workbench/contrib/terminal/browser/terminalGroupService.ts

```diff
@@ -191,6 +191,7 @@ export class TerminalGroupService extends Disposable implements ITerminalGroupSe
 	}
 
 	getFindState(): FindReplaceState {
+		KEYBINDING_CONTEXT_TERMINAL_FIND_VISIBLE.bindTo(this._contextKeyService).set(true);
```

Comment:
Can this be controlled by whatever handles showing/hiding of the find widget instead?

Reply:
that would be `focusFindWidget`, which I thought we were trying to avoid

Reply:
That's only when it gets focus isn't it? We need to differentiate these 2 contexts:

```
	FindVisible = 'terminalFindVisible',
	FindFocused = 'terminalFindFocused',
```